### PR TITLE
new privacy and ToU pages

### PR DESCRIPTION
**Pull request by dorianjp**
# Summary | Résumé

Trello:
https://trello.com/c/yRdx403p/787-implement-privacy-and-tou

* New pages for privacy and terms of use.

# Test instructions | Instructions pour tester la modification

Click on links in footer for privacy and terms of use in registration site.

### CODE REVIEW

**Comment by dorianjp** [comment]
[Implement Privacy and ToU](https://trello.com/c/yRdx403p/787-implement-privacy-and-tou)

File: register/templates/register/privacy.html [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+{% endblock %}
+
+
+
+
+
+
+
+
+
```
Comment (dsamojlenko) [inline register/templates/register/privacy.html:210]: Can't believe the linter doesn't catch this, but .. lots of whitespace at the end of this file.

File: register/templates/register/terms.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        {% trans 'you’re a person who’s responsible for making a poster for a place or event.' %}
+      </b>
+    </p>
+  <br>
+    <p>
+      <b>
+        {% trans 'You also agree that you’ll:' %}
+      </b>
+      <br>
+      <br>
```
Comment (dsamojlenko) [inline register/templates/register/terms.html:34]: Not crazy about all these `<br>`'s and double `<br>`'s for spacing ... we should either adjust the stylesheets, or consider bringing in some tailwind-style utility classes. I'm ok with revisiting that later.

**Review by dsamojlenko — APPROVED** [review]

File: register/templates/register/terms.html (hunk 2 above)
Comment (dorianjp) [reply register/templates/register/terms.html:34]: Yeah it's a funny spot, matching the design vs extra markup, or one off boutique css classes. I'm open to either, but the real answer lies in alignment in content spacing patterns in the design, H1/H2/p margins and padding are not consistent. Something worth looking into in a sweep for tech debt.

File: register/templates/register/privacy.html (hunk 1 above)
Comment (dorianjp) [reply register/templates/register/privacy.html]: fixed, thx

**Pull request merged**